apps/authentication-service/src/middleware/validateMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler } from "express";
+import { ObjectSchema } from "yup";
+
+const ReqValidateBody: ReqValidate = (schema) => async (req, res, next) => {
```

Comment:
Need tests for this

Reply:
I added a new e2e test. Should I create a unit test as well? @wesrisenmay-mx

Reply:
IMO yes. The e2e test is testing that validation is working for a specific endpoint. This should test that the validate middleware works.

Reply:
This still needs unit tests unless I'm missing them